test/w_http_client_integration_test.dart

```diff
@@ -106,5 +106,27 @@ void main() {
       Map data = JSON.decode(await response.asText());
       expect(data['headers']['x-configured'], equals('true'));
     });
+
+    group('should set the withCredentials flag', () {
```

Comment:
The `expect` statement is kind of hidden inside the `request.configure()` blocks below